test/TEST-74-AUX-UTILS/test.sh

```diff
@@ -36,6 +36,7 @@ test_append_files() {
     instmods vmw_vsock_virtio_transport
     instmods vsock_loopback
     instmods vmw_vsock_vmci_transport
+    inst_binary gcc
```

Comment:
As per the other PR this needs to move into a standalone test as this one runs in a container so it's not actually tested, it's skipped

Reply:
Test bails out early if it is running in container and that is true for the entire `TEST-74-AUX-UTILS.coredump.sh`. IOW, we don't test coredumpd and coredumpctl in containers and rightfully so. If we want to test coredump related components we need to run the test in a VM. I don't see how moving this new piece to a separate test case would help as I'd have to add the same condition at the beginning, i.e. if container then skip;

Reply:
Yeah that's what I meant - the coredump test needs to move to TEST-86-COREDUMP or so, configured to run as a vm

Reply:
OK, I will work with @mrc0mmand and we can do that but I think that should be done in a separate PR.